src/core.rs

```diff
@@ -0,0 +1,247 @@
+/// The main hash code used by Starknet.
+///
+/// Contains 251 bits of data and is generated by the pedersen hash function.
```

Comment:
```suggestion
/// Contains 251 bits of data and is generated by the [pedersen hash function].
///
/// [pedersen hash function]: https://docs.starkware.co/starkex-v3/crypto/pedersen-hash-function
```

Reply:
I didn't see this link being present anywhere. Maybe there could be a better link?